backend/fleet_management/factories.py

```diff
@@ -1,28 +1,28 @@
-import random
 import string
 from datetime import timedelta
 
-import factory
 import random
 from django.utils.timezone import now
-from factory import fuzzy
+from factory import fuzzy, DjangoModelFactory, Faker, LazyAttribute, lazy_attribute, \
```

Comment:
Looks like cosmetic change here. Don't see a reason for that, but I'm fine however it looks.

Reply:
`factory` was imported twice in two ways.
So eigher we go with `import factory` and we dont import `fuzzy` separetely or we do it as above.

Reply:
Don't recognize that as an error here, but I'm fine either way.